index.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable space-before-function-paren */
+/* eslint-disable brace-style */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable no-unused-vars */
```

Comment:
[Optional] As much as possible, kindly avoid disabling linter checks inline. This is because they most of the time suggest best coding practices.